docs/releases/1.10.txt

```diff
@@ -122,12 +122,12 @@ Minor features
   to prevent an issue where Safari caches redirects and prevents a user from
   being able to log out.
 
-* Added the optional ``backend`` argument to :func:`~django.contrib.auth.login`
+* Added the optional ``backend`` parameter to :func:`~django.contrib.auth.login`
```

Comment:
Refers to backend parameter:
https://github.com/lachilles/django/blob/a3ba2662cdaa36183fdfb8a26dfa157e26fca76a/django/contrib/auth/__init__.py#L118-L118